pyterrier/terrier/index_factory.py

```diff
@@ -60,17 +60,15 @@ def _load_into_memory(index, structures=['lexicon', 'direct', 'inverted', 'meta'
             }
 
         pindex = pt.java.cast("org.terrier.structures.IndexOnDisk", index)
-        load_profile = pindex.getIndexLoadingProfileAsRetrieval()
```

Comment:
@cmacdonald, from what I can tell, this line can be removed. But I'm also not certain about what it was trying to do originally, so I'd appreciate it if you could double check this one.

Reply:
Sorry, I'm still not seeing it -- can you point me to the specific line?

Reply:
looked at the wrong context, this occurrence is superfluous and can be removed.